hw3/multiplication.js

```diff
@@ -0,0 +1,12 @@
+function makeMultiplication(){
+  let counter=1;
+  return function(number){  
+    return counter *= number;
+  }  
+};
+const multi = makeMultiplication();
+
+module.exports = {
+    makeMultiplication,
```

Comment:
you exports 2 functions but use only one

Reply:
Fixed